lib/Subscriber/Core/RegistratorSubscriber.php

```diff
@@ -83,7 +84,7 @@ public function configureOptions(ConfigureOptionsEvent $event)
      *
      * @param HydrateEvent $event
      */
-    public function handleDefaultLocale(HydrateEvent $event)
+    public function handleDefaultLocale(AbstractMappingEvent $event)
```

Comment:
To be honest I really don't know anymore what the policy is with default
locales. The system was initially required a locale. I thought I then
implemented a default locale (which is the same behavior as the
PHPCR-ODM and the Doctrine extensions IIRC).

- The document should be persisted in the same locale that it was loaded
in, if no locale is given when persisting.

- If it is a new document, then the default locale should be used.
- If the document is not localized, then having a default locale does
  not matter.

On Thu, Apr 14, 2016 at 12:08:47AM -0700, Daniel Rotter wrote:
>    In [1]lib/Subscriber/Core/RegistratorSubscriber.php:
> 
>  > @@ -83,7 +84,7 @@ public function configureOptions(ConfigureOptionsEvent $event)
>  >       *
>  >       * @param HydrateEvent $event
>  >       */
>  > -    public function handleDefaultLocale(HydrateEvent $event)
>  > +    public function handleDefaultLocale(AbstractMappingEvent $event)
> 
>    I still don't like the concept of a default locale when persisting, and
>    Sulu already relies on not passing a locale in certain situations. I guess
>    this change would make all the subscribers checking on the locale changing
>    the behavior? Is this change necessary to these try/catches in the
>    MappingSubscriber?
> 
>    —
>    You are receiving this because you authored the thread.
>    Reply to this email directly or [2]view it on GitHub
> 
>    Reverse link: [3]unknown
> 
> References
> 
>    Visible links
>    1. https://github.com/sulu/sulu-document-manager/pull/72#discussion_r59671184
>    2. https://github.com/sulu/sulu-document-manager/pull/72/files/8e599707818caabeeea23d162b9a19cd78a0b234#r59671184
>    3. https://github.com/sulu/sulu-document-manager/pull/72/files/8e599707818caabeeea23d162b9a19cd78a0b234#r59671184